### DISCUSSION TITLE
How can I remove the black outline when i focus on the certain component.

### DISCUSSION THREAD

**Discussion by TammyCT** [twbs/bootstrap#30955] [General] [answered]
after I upgrade my project to bootstrap v4.5.0, I found that when i focus on some component, the black outline will show, but i don't want this. how can i remove the focus outline globally?
![QQ截图20200603115520](https://user-images.githubusercontent.com/31911204/83594252-58fc6b80-a591-11ea-9fc4-90c257ab2148.png)
![QQ截图20200603115734](https://user-images.githubusercontent.com/31911204/83594367-9eb93400-a591-11ea-9d73-a6edf857417a.png)

**Comment by patrickhlauke** [comment, marked as answer]
That's the browser's default focus outline, which is essential for keyboard users to be able to use a site. There are currently no plans for us to remove/suppress this.

Incidentally, you're looking at https://getbootstrap.com/docs/4.5/content/reboot/ which simply shows the default behavior of the browser for focus. 

https://getbootstrap.com/docs/4.5/components/forms/ shows slightly modified/softened focus styling that bootstrap uses.

http://www.outlinenone.com/

**Reply by TammyCT** [reply]
I do agree the outline is useful for the hotkey action, which is  really user-friendly. My project is in bootstrap v4.0.0-alpha.6, which is not outline when i focus on the component. Now I want to upgrade the boostrap to v4.5.0, but i want to consist the focus style as my old version looks like. How could I set my css to remove the outline, beacuse the outline is in all most every component such as button, input, grid cell etc.

**Reply by TammyCT** [reply]
i have set grid .ag-cell-focus style to outline:none, the issue can be fixed
